client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -129,6 +129,12 @@ export const ApplicationDetailDrawer: React.FC<
                 ? application.migrationWave.name
                 : t("terms.unassigned")}
             </Text>
+            <Title headingLevel="h3" size="md">
+              {t("terms.effort")}
+            </Title>
+            <Text component="small">
+              {application?.effort ? application.effort : t("terms.unassigned")}
```

Comment:
```suggestion
              {application?.effort ?? application.effort : t("terms.unassigned")}
```
...the optional chaining with nullish coalescing way